pkg/storage/fingerprint_writer.go

```diff
@@ -180,3 +182,73 @@ func (f *fingerprintWriter) stripTenantPrefix(key []byte) []byte {
 	}
 	return remainder
 }
+
+// FingerprintRangekeys iterates over the provided SSTs, that are expected to
+// contain only rangekeys, and maintains a XOR aggregate of each rangekey's
+// fingerprint.
+func FingerprintRangekeys(
+	ctx context.Context, cs *cluster.Settings, opts MVCCExportFingerprintOptions, ssts [][]byte,
+) (uint64, error) {
+	ctx, sp := tracing.ChildSpan(ctx, "storage.FingerprintRangekeys")
+	defer sp.Finish()
+	_ = ctx // ctx is currently unused, but this new ctx should be used below in the future.
+
+	iterOpts := IterOptions{
```

Comment:
Is it worth checking `len(ssts) == 0` here? The caller can pass an empty slice, and while I would expect the iterator handle this case, it might be prudent with an early return.

Reply:
yup, done.